pkg/label/label.go

```diff
@@ -102,6 +102,15 @@ const (
 	// AnnTiKVLastAutoScalingTimestamp is annotation key of tidbclusterto which ordinal is created by tikv auto-scaling
 	AnnTiKVLastAutoScalingTimestamp = "tikv.tidb.pingcap.com/last-autoscaling-timestamp"
 
+	// AnnTiDBConsecutiveScaleOutCount describes the least consecutive count to scale-out for tidb
+	AnnTiDBConsecutiveScaleOutCount = "tidb.tidb.pingcap.com/consecutive-scale-out-count"
+	// AnnTiDBConsecutiveScaleInCount describes the least consecutive count to scale-in for tidb
+	AnnTiDBConsecutiveScaleInCount = "tidb.tidb.pingcap.com/consecutive-scale-in-count"
```

Comment:
Can we store these fields in the status of tac?
What's the benefit with annotation? And why do we put them in tc insead of tac?

Reply:
If we stored the state in the tac annotation or status, there would be lots of caring about the tc status in tac controller logic. For example, changing the `tidbcluster Ref` or deleting the `TidbCluster`. More over, if I delete an existed tac, and recreated it soon, the auto-scaling status wouldn't be affected if we store is in Tc Annotations.

Reply:
updated.